tools/mo/unit_tests/mo/back/ChangeRandomUniformOutputType_test.py

```diff
@@ -31,15 +30,9 @@
                       *connect('convert', 'result'), ]
 
 
-@generator
 class ChangeRandomUniformOutputTypeTest(unittest.TestCase):
-    @generate(*[
-        ("FP16", np.float32, np.float16),
-        ("FP32", np.float16, np.float32),
-        ("FP32", np.float32, None),
-        ("FP32", np.int64, None)
-    ])
-    def test_change_random_uniform_output_type(self, ir_type, out_type, dst_type):
+    def test_change_random_uniform_output_type_case1(self):
```

Comment:
Can we use @pytest.mark.parametrize instead of copying the test with different parameters?

Reply:
@PRATHAM-SPS, can you please try to use pytests for tests that you are changing in your PR and use `parameterize` instead of tests-generator. Can you do this little experiment and share your experience?

Reply:
@rkazants @popovaan @mvafin I attempted to use pytest as recommended, but none of the test cases passed.

link:-  https://github.com/openvinotoolkit/openvino/compare/master...PRATHAM-SPS:openvino:pytest_testing?expand=1

Please advise if I'm doing it incorrectly.

Reply:
Hi @PRATHAM-SPS, I can give advice to take a look at our existing pytests here: https://github.com/openvinotoolkit/openvino/tree/master/tests/layer_tests

What kind or error are you getting?
I think the idea should be the same. Again. please update just one test and we will decide how to proceed.

@mvafin, @popovaan

Reply:
@PRATHAM-SPS Please try to apply my comments: https://github.com/openvinotoolkit/openvino/commit/3dadddf62e1d356a334786fbf15facf9c0f55069

With these corrections the test should work.

Reply:
@rkazants  E   ModuleNotFoundError: No module named 'openvino.tools.ovc' ,  how should i install it to resolve the issue? And yet another query? If pytest works properly, do I need to edit every file again? because it was suggested in the issue's description that it should be converted into a unittest's subTest, and it was confirmed that the file changes were acceptable.

Reply:
> ModuleNotFoundError: No module named 'openvino.tools.ovc'
Please merge the latest master. We removed OVC dependency from MO, so the issue should be fixed.

Reply:
@PRATHAM-SPS, did you experiment with using `pytest` just for one file instead of `generator`?

Reply:
Hello @rkazants , pytest runs without an issue. One file was tested, and it passed every test case using Pytest

Here is the file:-https://github.com/openvinotoolkit/openvino/compare/master...PRATHAM-SPS:openvino:pytest_testing

and the results I posted right above your comment.